hw2/slavrom.js

```diff
@@ -0,0 +1,135 @@
+// 1
+
+randomNumbers = (arr, num) => {
+    let result = [];
+    if (num > arr.length) {
+        return 'Taken more elements than available !';
+    } else {
+        while (num--) {
+            let x = arr[Math.floor(Math.random() * arr.length)];
+            result.push(x);
+        }
+    }
+    return `${result.length} random elements from array: ${result}`;
+}
+
+let numbers = [12, 34, 23, 56];
+console.log(randomNumbers(numbers, 3));
```

Comment:
the numbers are repeating sometimes. Would be nice to think of solution to avoid the duplicates